### DISCUSSION TITLE
Can not save PS game.

### DISCUSSION THREAD

**Discussion by kidbimilky** [eduardofilo/RG350_adam_image#280] [General]
I played Suikoden II (PSX) on v.2.1, and can save game in first but next time, the rg350 automaticly reset and Iost the saved game. You can fix in future?

**Comment by eduardofilo** [comment]
I have tried that game and have had no problems. I have saved and loaded the savestate several times, even closing the emulator and restarting the console some of the times. 0 problems. Try another ROM of the same game or install another PSX BIOS.
